src/app/calender/page.tsx

```diff
@@ -1,6 +1,6 @@
 "use client";
 
-import CalenderPage from "@/templates/calender";
+import CalenderPage from "@/templates/calender/layouts";
```

Comment:
```suggestion
import CalendarPage from "@/templates/calendar/layouts";
```
캘린더는 calendar가 맞는 단어라고 합니다👍
다 수정하기 빡세겠네요ㄷ

Reply:
220개를 수정했습니다^^..